src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -157,4 +157,12 @@ public void testEmptyCatchBlock() throws Exception {
         testByConfig("EmptyCatchBlock/patchedline/defaultContextConfig.xml");
         testByConfig("EmptyCatchBlock/context/defaultContextConfig.xml");
     }
+
+    @Test
+    @Ignore("until https://github.com/checkstyle/patch-filters/issues/152")
```

Comment:
Why is it ignored?

Reply:
https://github.com/checkstyle/patch-filters/pull/154#discussion_r461371947

Reply:
@HuGanghui please add this case to the corresponding group of patterns if you haven't done it yet.